internal/manager/config_validation.go

```diff
@@ -122,6 +126,22 @@ func (c *Config) validateFallbackConfiguration() error {
 	return nil
 }
 
+func (c *Config) validateGatewayDiscovery() error {
+	// Skip validation if gateway discovery is not enabled.
+	if _, ok := c.KongAdminSvc.Get(); !ok {
+		return nil
+	}
+	// REVIEW: Should the reconciliation interval has a miniumum value and should it be the same as default interval?
```

Comment:
I believe the interval should have a minimum. I would suggest something like 3s. WDYT?

Reply:
3s that is same as default config sync interval seems good.